node/src/actors/chain_manager/transaction_factory.rs

```diff
@@ -198,7 +213,24 @@ mod tests {
         own_pkh: PublicKeyHash,
         all_utxos: &UnspentOutputsPool,
     ) -> Result<Transaction, NoMoney> {
-        let vtt_tx = build_vtt(outputs, fee, own_utxos, own_pkh, all_utxos)?;
+        let timestamp = get_timestamp() as u64;
```

Comment:
We don't need the real timestamp in tests, just something > 0.

```suggestion
        let timestamp = 1;
```